phone_book/book_generator.py

```diff
@@ -0,0 +1,18 @@
+import csv
+
+def add_phone():
+    with open('phone_book.csv', 'a', newline='') as file:
+        name_records = ('id', 'name', 'surname', 'birth', 'employment', 'phone')
+        new_record = []
+        for i in range(len(name_records)):
+            print(f'Enter {name_records[i]}: ', end='')
```

Comment:
```suggestion
        for head in name_records:
            print(f'Enter {head}: ', end='')
```
Мне кажется так чуточку проще.

Reply:
Точно )